lvsfunc/misc.py

```diff
@@ -415,6 +417,50 @@ def upscale_chroma(n: int, f: vs.VideoFrame) -> vs.VideoNode:
     return cast(F, inner)
 
 
+def colored_clips(amount: int,
+                  max_hue: int = 300,
+                  rand: bool = True,
+                  seed: Optional[Union[bytearray, bytes, float, int, str]] = None,
+                  **kwargs: Any
+                  ) -> List[vs.VideoNode]:
+    """
+    Returns a list of BlankClips with unique colors (equally spaced in the HSL's hue domain) in sequential or random order.
+    Useful maybe for comparison functions or just for getting multiple uniquely colored BlankClips for testing purposes.
+
+    Will always return a pure red clip in the list as this is the RGB equivalent of the lowest HSL hue possible (0).
+
+    Written by Dave <orangechannel@pm.me>.
+
+    :param amount:  Number of ``vapoursynth.VideoNode``s to return
+    :param max_hue: Maximum hue (0 < hue <= 360) in degrees to generate colors from (uses the HSL color model).
+                    Setting this higher than ``315`` will result in the clip colors looping back towards red and is not recommended for visually distinct colors.
+                    If the `amount` of clips is higher than the `max_hue` expect there to be identical or visually similar colored clips returned (Default: 300)
```

Comment:
Does it make sense to have such a parameter in the first place? If you consider the hue plane to be cyclic from 0 to 360 (which it is), you just divide 360 into evenly spaced segments.

```py
>>> (lambda count: [360 / (i + 1) for i in range(count)])(6)
[360.0, 180.0, 120.0, 90.0, 72.0, 60.0]
```

Reply:
Try this with 16 clips though (which is what I did to test a mosaic compare). The end value is 337.5 which is basically red. I know by limiting it to 300 there's less space between the colors too which makes them *all* look more similar but I thought it'd be better to stop the colors from bleeding into the red section again. This lambda might make more sense though.